pkg/webhooks/workload_webhook.go

```diff
@@ -96,8 +105,17 @@ func ValidateWorkload(obj *kueue.Workload) field.ErrorList {
 	var allErrs field.ErrorList
 	specPath := field.NewPath("spec")
 
+	variableCountPosets := 0
 	for i := range obj.Spec.PodSets {
-		allErrs = append(allErrs, validatePodSet(&obj.Spec.PodSets[i], specPath.Child("podSets").Index(i))...)
+		ps := &obj.Spec.PodSets[i]
+		allErrs = append(allErrs, validatePodSet(ps, specPath.Child("podSets").Index(i))...)
+		if pointer.Int32Deref(ps.MinCount, ps.Count) != ps.Count {
+			variableCountPosets++
+		}
+	}
+
+	if variableCountPosets > 1 {
+		allErrs = append(allErrs, field.Invalid(specPath.Child("podSets"), variableCountPosets, "at most one podSet can have variable count"))
```

Comment:
```suggestion
		allErrs = append(allErrs, field.Invalid(specPath.Child("podSets"), variableCountPosets, "at most one podSet can use minCount"))
```

Also please add it to the API documentation (as part of the comment in the field).